bin/si-api-test/tests/7-emulate_authoring.ts

```diff
@@ -0,0 +1,462 @@
+// deno-lint-ignore-file no-explicit-any
+import assert from "node:assert";
+import { SdfApiClient } from "../sdf_api_client.ts";
+import {
+    runWithTemporaryChangeset,
+    sleep,
+    sleepBetween,
+    createComponent,
+    createAsset,
+    updateAssetCode,
+    nilId,
+    getQualificationSummary,
+    getPropertyEditor,
+    attributeValueIdForPropPath,
+    setAttributeValue,
+} from "../test_helpers.ts";
+import { ulid } from "https://deno.land/x/ulid@v0.3.0/mod.ts";
+
+export default async function emulate_authoring(sdfApiClient: SdfApiClient) {
+    await sleepBetween(0, 750);
```

Comment:
nitpick: personally I'd remove this line, it just slows down the tests.

If we are looking to stagger the tests it should be in the harness above, not the test itself.